.git-blame-ignore-revs

```diff
@@ -1,3 +1,6 @@
 # .git-blame-ignore-revs
 # autopep8 python PR3268
 b9dbef317197b2f450b3e963fb4744cc4dc5e087
+
+# clang-format stuffer
+94515439505c7dc70809d70baec3a905f8711dc7
```

Comment:
Does this file need a newline?

Reply:
I guess not. It didn't have one before.

Reply:
I looks like github didn't mark the newline as missing in the original commit, so I think it did previously have the newline: https://github.com/aws/s2n-tls/pull/3268/files#diff-f247fbfbe928b907db42554d0b3006b28dd11c25a59be031abda73b11a2c934a

You should always add that final newline. I've never seen an "unnecessary" newline break a tool, but I've definitely seen missing newlines break tools. That's why Github puts the red mark there to point it out :)

Reply:
I'll move this into a separate PR after this one merges.